data/json/items/armor/head_attachments.json

```diff
@@ -211,7 +211,7 @@
     "type": "TOOL_ARMOR",
     "category": "armor",
     "name": { "str_sp": "night vision goggles" },
-    "description": "Last generation double-tube night vision device, designed to be mounted to a helmet.  Currently raised up and out of the way.",
+    "description": "Last generation double-tube night vision device, designed to be (i)nserted on a compatible helmet.  Currently raised up and out of the way.",
```

Comment:
I feel like mentioning inserting as a game mechanic isn't necessary here. I think "mounting" already covers it.